examples/custom-types/expected/Operations/MyBenSampoEnumQuery.php

```diff
@@ -36,6 +36,11 @@ public static function document(): string
         }';
     }
 
+    public static function config(): string
+    {
+        return '/home/bfranke/projects/sailor/examples/custom-types/sailor.php';
```

Comment:
It might be problematic for deployment to have the absolute path here. Perhaps we should rather use a relative path from this file to the configuration file.

```suggestion
        return __DIR__ . '/../../../sailor.php';
```

Reply:
@spawnia I'd expect the path to be hardcoded here during code generation since codegen command knows what configuration file it uses.